addons/account/views/account_account_views.xml

```diff
@@ -93,7 +94,8 @@
             <field name="model">account.account</field>
             <field name="arch" type="xml">
                 <list editable="top" create="1" delete="1" multi_edit="1" string="Chart of accounts"  open_form_view="True">
-                    <field name="code"/>
+                    <field name="placeholder_code" column_invisible="1"/>
+                    <field name="code" string="Code" widget="char_with_placeholder_field" options="{'placeholder_field': 'placeholder_code'}"/>
```

Comment:
Shouldn't there be a default placeholder here too? (like in the form view)

Reply:
I'll ask Benj.

Reply:
(Benj says no need, it hasn't bothered anyone so far)